query/src/api/http/v1/logs.rs

```diff
@@ -38,15 +39,27 @@ pub async fn logs_handler(
     Ok(data)
 }
 
-async fn select_table(sessions: &Arc<SessionManager>) -> Result<String> {
+async fn select_table(sessions: &Arc<SessionManager>) -> Result<Body> {
     let session = sessions.create_session("WatchLogs")?;
     let query_context = session.create_context().await?;
+    let mut tracing_table_stream = execute_query(query_context).await?;
 
-    let tracing_table_stream = execute_query(query_context).await?;
-    let tracing_logs = tracing_table_stream
-        .collect::<Result<Vec<DataBlock>>>()
-        .await?;
-    Ok(format!("{:?}", tracing_logs))
+    let stream = async_stream::try_stream! {
+        match tracing_table_stream.next().await {
+            Some(res) => {
+                let block = res?;
+                yield format!("{:?}", block);
+            },
+            None => return,
+        }
+
+        while let Some(res) = tracing_table_stream.next().await {
+            let block = res?;
+            yield format!(", {:?}", block);
+        }
```

Comment:
Why does it need a `match` and a `while`? 
A single `while` is not enough? 🤔

Reply:
The final data will be separated by commas, which is the original behavior.

Maybe this comma separator doesn't make sense? 😂

```rust
let tracing_logs = tracing_table_stream
        .collect::<Result<Vec<DataBlock>>>()
        .await?;
Ok(format!("{:?}", tracing_logs))
```

Reply:
I did not see the comma...

Apologize!

/lgtm

Reply:
How it compare with 
```
if i > 0 {
  yield format!(", {:?}", block);
} else {
  i += 1;
  yield format!("{:?}", block);
}
```